cmd/runtimetest/main.go

```diff
@@ -588,6 +588,8 @@ func validate(context *cli.Context) error {
 		return err
 	}
 
+	platform := context.String("platform")
```

Comment:
I mean we should change:  
```
g.SetProcessArgs([]string{"/runtimetest"})
```
to
```
g.SetProcessArgs([]string{"/runtimetest", "platform", platform})
```
since you add a platform option to runtimetest command.